pkg/k8shandler/rbac.go

```diff
@@ -44,8 +42,6 @@ func (er *ElasticsearchRequest) CreateOrUpdateRBAC() error {
 		),
 	)
 
-	addOwnerRefToObject(elasticsearchRole, owner)
```

Comment:
why do we not have a `dpl.AddOwnerRefTo` call here?

Reply:
If you expand the folded code above, you will see that this reference is applied to a cluster-scoped resource, i.e. the ClusterRole. Exactly this kind of ownerReferences are the source of the unwanted garbage collection.

Reply:
ack, it wasn't clear from the description that that was the source of this... applying an owner ref to cluster-scoped resources